### PR TITLE
WIP Maresb/rowan branch3

### PR DESCRIPTION
**Pull request by rowan-schaefer**
**What is this PR about?**
...

**Checklist**
+ [ ] Explain important implementation details 👆
+ [ ] Make sure that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html).
+ [ ] Link relevant issues (preferably in [nice commit messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html))
+ [ ] Are the changes covered by tests and docstrings?
+ [ ] Fill out the short summary sections 👇

## Major / Breaking Changes
- ...

## Bugfixes / New features
- ...

## Docs / Maintenance
- ...

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ vscode:
   extensions:
     - eamodio.gitlens
     - ms-python.python
+    # hello!!!!!!!!!!!!!!!!😃 ⭐ ️️
```
Comment (canyon289) [inline .gitpod.yml:18]: Thanks for pr. Is this ready to review?

**Comment by canyon289** [comment]
As always PRs are welcome. An informative title would be helpful. Whats the intent of this one? Its not communicated in title of `Maresb/rowan branch3`

File: .gitpod.yml (hunk 1 above)
Comment (rowan-schaefer) [reply .gitpod.yml:18]: Sorry - I did not mean to submit this pr! 🤦‍♀️ We're messing with gitpod

Comment (canyon289) [reply .gitpod.yml:18]: All good :D

**Comment by canyon289** [comment]
marked as WIP. Looking forward to the final result

**Pull request closed without merging**